docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1384,6 +1384,115 @@ containers:
 {% endnavtab %}
 {% endnavtabs %}
 
+#### CSI
+Other than using Alluxio FUSE daemon, you could also use CSI to mount the Alluxio FileSystem into application containers.
+
+In order to use CSI, you need a Kubernetes cluster with version at least 1.17, 
+with [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) enabled in API Server.
+
+**Step 1: Customize configurations and generate templates**
+
+You can either use the default CSI configurations provided in
+[here](https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/values.yaml)
+under the csi section, or you can customize them to make it suitable for your workload.
+Here are some common properties that you can customize:
+<table class="table table-striped">
+  <tr>
+    <th>property name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>alluxioPath</td>
+    <td>The path in Alluxio which will be mounted</td>
+  </tr>
+  <tr>
+    <td>mountPath</td>
+    <td>The path that Alluxio will be mounted to in the application container</td>
+  </tr>
+  <tr>
+    <td>javaOptions</td>
+    <td>The customized options which will be passes to fuse daemon</td>
+  </tr>
+  <tr>
+    <td>mountOptions</td>
+    <td>Alluxio Fuse mount options</td>
+  </tr>
+</table>
+
+Then please use `helm-generate.sh` (see [here](https://github.com/Alluxio/alluxio/tree/master/integration/kubernetes#generate-kubectl-yaml-templates-from-helm-chart) for usage)
```

Comment:
I still need to manually generate templates? lol, can we have a command line example if needed? or if it's already included in the alluxio tarball, then we can just use those